pontoon/base/utils.py

```diff
@@ -4,7 +4,6 @@
 import os
 import re
 
-import pytz
```

Comment:
I think this is the only place where we use `pytz` directly, so could drop it from `requirements/default.in`?